apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -2317,7 +2317,7 @@ suite('thread_list_ui', function() {
 
       Drafts.on.withArgs('saved').yield(threadDraft);
 
-      assert.isTrue(InboxView.notifyAboutSavedDraft);
+      assert.equal(InboxView.notifyAboutSavedDraftWithId, threadDraft.id);
```

Comment:
I don't really like this check. I think we should never check for the value of this internal property, only check the _effect_ of this property. What do you think ?